recipes/qurro/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "qurro" %}
+{% set version = "0.6.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9c911927525ed823d65cc017beb1ba8ab9dc7226986141bcb998233915c1620d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - qurro=qurro.scripts._plot:plot
+    - q2-qurro=qurro.q2.plugin_setup:plugin
+  skip: True  # [py<35 or win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython
+    - numpy >=1.12.0
+  run:
+    - python
+    - setuptools
+    - cython
```

Comment:
Is `cython` or `setuptools` required to run the package? 

I'm not even really seeing `cython` needed explicitly in the `setup.py` to build anything, just a note in the docs. Can you explain why it is needed? Here you need it in `host` because of `setup_requires` in the `setup.py`, so definitely leave it in `host`. It's more of a question of what is going on upstream.

Reply:
I've removed `cython` and `setuptools` from the `run:` section. `cython` is required to install `biom-format`, which is a dependency of Qurro -- as I remember, we started seeing errors recently when `biom-format` 2.1.8 was released that were resolved by installing `cython` before attempting to install `biom-format`.